public/app/plugins/datasource/prometheus/querybuilder/aggregations.ts

```diff
@@ -126,14 +141,18 @@ function getAggregationWithoutRenderer(aggregation: string) {
 /**
  * Very simple poc implementation, needs to be modified to support all aggregation operators
  */
-function getAggregationExplainer(aggregationName: string, mode: 'by' | 'without') {
+function getAggregationExplainer(aggregationName: string, mode: 'by' | 'without' | '') {
```

Comment:
nit: Maybe making the mode optional as `mode?: 'by' | 'without'` would be a bit nicer.

Reply:
well figured the mode is by, without or empty (ie no extra clause), if mode is optional, might signify that it has some default?

Reply:
well figured the mode is by, without or empty (ie no extra clause), if mode is optional, might signify that it has some default?

Reply:
yea maybe mode is bad name for that param, maybe "labelClause" or something is better